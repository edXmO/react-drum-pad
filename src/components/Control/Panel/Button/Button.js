import React from 'react';

const Button = ({ text, type }) => {
    return (
        <div className='btn-panel__box'>
            <button className='btn-panel__btn'></button>
            {text}
        </div>
    )
}

export default Button;