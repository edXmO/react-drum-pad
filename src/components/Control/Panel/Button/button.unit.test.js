import React from 'react';
import renderer from 'react-test-renderer';

// Component
import Button from './Button';

describe('should render without crashing', () => {
    let component = renderer.create(
        <Button type={'Volume'} text={'+VOL'} />
    );
    it('should render without crashing', () => {
        expect(component.toJSON()).toMatchSnapshot();
    });
});