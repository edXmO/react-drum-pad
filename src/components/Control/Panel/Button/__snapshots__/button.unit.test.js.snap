// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing should render without crashing 1`] = `
<div
  className="btn-panel__box"
>
  <button
    className="btn-panel__btn"
  />
  +VOL
</div>
`;
